presto-raptor/src/main/java/com/facebook/presto/raptor/backup/BackupConfig.java

```diff
@@ -58,4 +59,16 @@ public BackupConfig setProvider(String provider)
         this.provider = provider;
         return this;
     }
+
+    public int getBackgroundBackupThreads()
+    {
+        return backgroundBackupThreads;
+    }
+
+    @Config("backup.background-threads")
```

Comment:
I'm thinking this should be `backup.threads` or `backup.max-threads` since "background" seems redundant